src/constants/constants.ts

```diff
@@ -1,3 +1,8 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
It'd be great to add a gulp hygiene task like we have for ADS that checks for these as a precommit hook

Reply:
We do - checking in changes to a file will flag missing copyright (and just a space before it, as I've discovered a bunch).

Reply:
Oh sorry - wrong repo. Ignore my comment.